lib/src/json_schema/json_schema.dart

```diff
@@ -124,7 +125,7 @@ class JsonSchema {
     List<CustomVocabulary> customVocabularies,
     Map<String, Map<String, SchemaPropertySetter>> customVocabMap,
     Map<String, ValidationContext Function(ValidationContext context, String instanceData)> customFormats = const {},
-  }) {
+  }) : _schemaMap = null {
```

Comment:
I had forgotten about this initialization trick in dart.

Reply:
Right?! I had to ping Michael to be like "What the heck is that?"